packages/secure-store/legacyUnlock.ts

```diff
@@ -0,0 +1,41 @@
+import { ariesAskar } from '@hyperledger/aries-askar-shared'
+import * as SecureStore from 'expo-secure-store'
+
+const STORE_KEY_LEGACY = 'wallet-key' as const
+const STORE_KEY_RAW = 'paradym-wallet-key-raw' as const
```

Comment:
paradym specific identifier

Reply:
Yes, but this need to stay the same as this it the legacy unlock (currnetly used by paradym wallet). We can't just change the key